onnx/defs/quantization/defs.cc

```diff
@@ -145,9 +145,9 @@ y = saturate (round (x / y_scale) + y_zero_point)
 
 ONNX_OPERATOR_SET_SCHEMA(
     DynamicQuantizeLinear,
-    11,
+    13,
     OpSchema()
-        .SetDoc(DynamicQuantizeLinear_ver11_doc)
+        .SetDoc(DynamicQuantizeLinear_ver13_doc)
         .Input(0, "x", "Input tensor", "T1")
         .Output(0, "y", "Quantized output tensor", "T2")
         .Output(1, "y_scale", "Output scale. It's a scalar, which means a per-tensor/layer quantization.", "tensor(float)")
```

Comment:
We'd better to make DynamicQuantizeLinear to support per-channel too.

Reply:
It looks hard to implement with FunctionBody because there is no way to generate axes attribute for ReduceMin correctly.